pylint/checkers/similar.py

```diff
@@ -422,12 +422,12 @@ def get_map_data(self):
         """ Passthru override """
         return Similar.get_map_data(self)
 
-    @classmethod
-    def reduce_map_data(cls, linter, data):
+    def reduce_map_data(self, linter, data):
         """Reduces and recombines data into a format that we can report on
 
         The partner function of get_map_data()"""
         recombined = SimilarChecker(linter)
+        recombined.min_lines = self.min_lines
```

Comment:
This probably needs an explanatory comment.

# Copy down relevant options

Reply:
That's right, done.